src/extras/render-passes/render-pass-camera-frame.js

```diff
@@ -142,11 +144,19 @@ class RenderPassCameraFrame extends RenderPass {
 
     needsReset(options) {
         const currentOptions = this.options;
+
+        // helper to compare arrays
+        const arraysNotEqual = (arr1, arr2) => arr1 !== arr2 &&
+            (!(Array.isArray(arr1) && Array.isArray(arr2)) ||
+            arr1.length !== arr2.length ||
+            !arr1.every((value, index) => value === arr2[index]));
```

Comment:
Did you forget about this:
https://github.com/playcanvas/engine/blob/main/src/core/array-utils.js#L4

Reply:
I spent few minutes looking for this and didn't find it .. I though I removed it in 2.0 :)
Will adjust, thanks!